src/Nulls.jl

```diff
@@ -150,6 +150,25 @@ julia> coalesce.([null, 1, null], [0, 10, 5])
 coalesce(x) = x
 coalesce(x, y...) = ifelse(x !== null, x, coalesce(y...))
 
+"""
+    levels(x)
+
+Return a vector of unique values which occur or could occur in collection `x`,
+omitting `null` even if present. Values are returned in the preferred order
+for the collection, with the result of [`sort`](@ref) as a default.
+
+Contrary to [`unique`](@ref), this function may return values which do not
+actually occur in the data, and does not preserve their order of appearance in `x`.
+"""
+function levels(x)
```

Comment:
should we restrict the argument to `AbstractArray`?

Reply:
I've considered it, but I saw no reason to reject iterables since `unique` supports them.

Reply:
One argument in favor of allowing iterables is that `levels(Nulls.skip(x))` wouldn't work otherwise.

Reply:
Since `levels` skip nulls, you shouldn't need to skip them. But yeah, if somebody passed your function an iterator in order to skip nulls and you call `levels` on that, it's nice that it works.